packages/javascript/bh-shared-ui/src/components/HelpTexts/CodeController/CodeController.tsx

```diff
@@ -0,0 +1,155 @@
+// Copyright 2024 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { Button, Typography } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { PropsWithChildren, useRef, useState } from 'react';
+import clsx from 'clsx';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faAlignJustify, faCopy } from '@fortawesome/free-solid-svg-icons';
+import { copyToClipboard } from '../../../utils/copyToClipboard';
+
+export const useStyles = makeStyles((theme) => ({
```

Comment:
The updated styles for this look nice! A lot clearer in the intention and more noticeable than the initial proof of concept.